rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -21,7 +21,10 @@ def test_executable():
     output_files = '@RCLCPP_EXAMPLES_EXPECTED_OUTPUT@'.split(';')
     for i, (exe, output_file) in enumerate(zip(executables, output_files)):
         name = 'test_executable_' + str(i)
-        handler = create_handler(name, launch_descriptor, output_file)
+        # The first executable is taken to be the test program (the one whose
+        # output check can make the decision to shut everything down)
+        test_process = True if i == 1 else False
```

Comment:
For for simplicity of boolean logic that equates to:

    test_process = (i == (len(executables) - 1))